virtcontainers/qemu.go

```diff
@@ -423,6 +423,17 @@ func (q *qemu) setupTemplate(knobs *govmmQemu.Knobs, memory *govmmQemu.Memory) g
 	return incoming
 }
 
+func (q *qemu) setupFileBackedMem(knobs *govmmQemu.Knobs, memory *govmmQemu.Memory) {
+	knobs.FileBackedMem = true
+	knobs.FileBackedMemShared = true
+	if q.config.FileBackedMemRootDir == "" {
+		//By default let us use /dev/shm
+		memory.Path = "/dev/shm"
```

Comment:
- I'd be tempted to make a `const defaultFileBackedMemRootDir = "/dev/shm"` at the top-level for this.
- Could you contrive a new unit test for this change?

Reply:
I suppose the shm mount should be present in most linux distributions. But we should atleast check for this early on.

Reply:
@jodh-intel have moved it up to the top. Let me see about tests for this.